docs/ui/viewmodel.md

```
@@ -0,0 +1,108 @@
+await viewModel.login('email', 'pw');
+expect(viewModel.state, isA<AsyncData>());
+```
+
+---
+
+## 🔁 관련 문서 링크
+
+- [usecase.md](../logic/usecase.md): 결과 전달 및 예외 처리 흐름
+- [state.md](state.md): 상태 모델 정의 가이드
+- [screen.md](screen.md): UI에서 상태 처리 방식
+- [../arch/folder.md](../arch/folder.md): 전체 파일 구성 구조
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)